### DISCUSSION TITLE
Self host login issue

### DISCUSSION THREAD

**Discussion by Evsio0n** [supabase/supabase#23246] [General]
Hey, 
I am a bit confused about supabase login.

On this video,
https://www.youtube.com/watch?v=wyUr_U6Cma4&ab_channel=activenode

I can see through old supabase version they did has a login UI for the studio `/sign-in` 
<img width="1258" alt="Screenshot 2024-04-25 at 6 13 42 AM" src="https://github.com/supabase/supabase/assets/29672485/80b79145-b953-4e13-ad55-1859dc7d758b">

However, I tried but not working.

And My part of authentication has no configurable options like providers, etc.
Left side was supabase.com and right side was self host.

<img width="627" alt="Screenshot 2024-04-25 at 6 10 31 AM" src="https://github.com/supabase/supabase/assets/29672485/a9084f47-4423-43a4-8a17-a749a2aabad3">

Is there a option or  possible to login supabase on self  hosted version, more like the cloud experience and stop using web basic auth?

<img width="499" alt="Screenshot 2024-04-25 at 6 17 15 AM" src="https://github.com/supabase/supabase/assets/29672485/e57f4b67-146e-49bc-9a18-851d27209be9">

Plus, I see many people use supabase as auth provider. Do they use cloud hosted version?

**Comment by GaryAustin1** [comment]
https://supabase.com/docs/guides/self-hosting/docker#accessing-supabase-dashboard
Discusses the login method for self hosted.
The auth settings are all done thru config files for self-hosted and not in the UI.
https://supabase.com/docs/guides/self-hosting

**Reply by Evsio0n** [reply]
Thanks for prompt response.
I only see the basic auth. Any solution to get authenticated  via oidc, google, etc?

**Reply by Evsio0n** [reply]
https://supabase.com/docs/guides/self-hosting/auth/config
Selfhost with docker dost not has cli of `supabase init`

**Reply by GaryAustin1** [reply]
Is there a sample config.toml that comes after your install?
That I believe is where you set things and I found one in the repository that seems to have some auth example settings include a 3rd party provider.
https://github.com/supabase/supabase/blob/09c397f331f0b7b6d8595fa80328aeeee4758be0/supabase/config.toml#L4

![image](https://github.com/supabase/supabase/assets/54564956/19b42855-2314-404a-bc19-2d55ac4a58b2)

The auth repository has what should be the available settings.
https://github.com/supabase/auth?tab=readme-ov-file#external-authentication-providers

I don't do self-hosting, so just have the docs.  You can also search the discussions here.

**Comment by gbcatrinoiu** [comment]
@Evsio0n did you find a way to switch away from the basic login?
